lib/scopes/form526_submission_state.rb

```diff
@@ -4,6 +4,7 @@ module Scopes
   module Form526SubmissionState
     extend ActiveSupport::Concern
 
+    # rubocop:disable Metrics/BlockLength
```

Comment:
Use of inline disable approved by Eric Boehs https://dsva.slack.com/archives/CBU0KDSB1/p1724785364018449?thread_ts=1724078250.589569&cid=CBU0KDSB1